r-pkg/tests/testthat/test-repo_characteristics.R

```diff
@@ -35,7 +35,7 @@ test_that('R CMD check should not return any unexpected errors, warnings, or not
     testthat::skip_on_cran()
     
     # Check the package
-    x <- devtools::check(pkg = '../../../uptasticsearch'
+    x <- devtools::check(pkg = '../../../r-pkg'
```

Comment:
I'll pull and check. But yeah it looks right. I forgot to change this when we removed the intermediate `uptasticsearch` directory.